tests/run/datetime_members.pyx

```diff
@@ -20,31 +21,35 @@ def test_date(int year, int month, int day):
            o.month == date_month(o), \
            o.day == date_day(o)
 
-def test_datetime(int year, int month, int day, 
-                  int hour, int minute, int second, int microsecond):
+def test_datetime(int year, int month, int day, int hour,
+                  int minute, int second, int microsecond, int fold):
     '''
-    >>> test_datetime(2012, 12, 31, 12, 30, 59, 12345)
-    (True, True, True, True, True, True, True)
+    >>> test_datetime(2012, 12, 31, 12, 30, 59, 12345, 0)
+    (True, True, True, True, True, True, True, True)
```

Comment:
```suggestion
    >>> test_datetime(2012, 12, 31, 12, 30, 59, 12345, 0)
    (True, True, True, True, True, True, True, True)
    >>> test_datetime(2012, 12, 11, 12, 30, 59, 3322, 1 if sys.version_info >= (3, 7) else 0)
    (True, True, True, True, True, True, True, True)
```

Reply:
```suggestion
    >>> test_datetime(2012, 12, 31, 12, 30, 59, 12345, 1)
    (True, True, True, True, True, True, True, True)
```
We use `datetime_fold` and the `datetime.fold` property, both call `PyDateTime_DATE_GET_FOLD` which we've backported to return 0 when necessary. So in the case of Python > 3.7 both would return 0 and become True even when we pass 1 as the argument.

Reply:
I think it's more explicit like this. We shouldn't test something that does something else under the hood.